ui/pages/confirmations/components/transaction-alerts/transaction-alerts.test.js

```diff
@@ -558,3 +563,47 @@ describe('TransactionAlerts', () => {
     });
   });
 });
+
+describe('Smart Transactions Migration Alert', () => {
+  it('should show smart transactions banner when shouldShowSmartTransactionsMigrationAlert is true', () => {
```

Comment:
Minor, we're referencing the `shouldShowSmartTransactionsMigrationAlert` selector in the test names but in the context of the test we don't know anything about that as we're just setting state.

For example, could this just be `shows if alert not closed in state`?

Reply:
I have updated this test. Thanks this is good feedback.